packages/docs/src/routes/docs/menu.md

```diff
@@ -23,6 +21,8 @@
 
 ## Routing
 
+- [Qwik City](</docs/(qwikcity)/qwikcity/index.mdx>)
+- [Project structure](</docs/(qwikcity)/project-structure/index.mdx>)
```

Comment:
you know, the whole doc frankly has issues, because it starts with "how to get started" via the CLi command, so it's really confusing to have this show up right after the "getting started" doc by it then telling you to get started again (and a little different)

Technically project structure and routing are pretty synonymous, so moved it here. I'm open to more changes though, I think the doc could use a revamp in general, if not at least just removing the top section.

To me though it feels like this should be a Qwik City section, instead of called Routing, so that wsa the other idea - moving toward that

We could also rename the section `Qwik City (Routing)` but also a little confusing, esp bc peopel think of it as a "meta framework" which is not _just_ routing in many peoples minds

So anyway, broad topic, this was a quick fix for the main issue that our get started path was confusing - basically telling you to get started twice and didn't really make sense, so the goal was to just make the first touch less confusing for new uesrs more than anything

Reply:
totally see your point 🙏 thanks a lot for the in depth explanation!